### ISSUE TITLE
Large performance gap between my evaluation and results in the technical report

### ISSUE THREAD

**Issue by madorism** [Google-Health/medgemma#21]
Hello,

I am currently evaluating the new MedGemma model on MIMIC-CXR report generation, but I observed a large performance gap compared to the results reported in the technical report.

I have tried to replicate the data cleaning and preprocessing described in the paper as closely as possible： Only use examples in  the test set with non-empty findings, impressions, and indication sections. Use pretrained model instead of IT model for report generation task. Follow the prompt in the technical report.

Despite following these steps, my results differ significantly, where I got lower than 24 for radgraph F1 score.
Also I noted many repetition and  even nonsense content.

Could you please clarify:
Which exact test set was used for the reported evaluation?
Were there any additional preprocessing steps not mentioned in the technical report?
Is it possible to access the exact evaluation split you used?

Thank you for your time and for releasing this work — it’s very valuable for the medical AI community.
Best regards,

**Comment by LakshmiKalaKadali** [comment]
Hi @madorism, 

Thank you for your interest in MedGemma and for taking the time to share your evaluation details on the MIMIC-CXR report generation task. To answer your questions about evaluation splits and preprocessing steps, please check the data splits and training details mentioned in our current [MedGemma 1.5 Technical Report ](https://arxiv.org/abs/2604.05081).  As your inquiry relates to our earlier version of the model, could you please validate your results on `Medgemma 1.5` model  and if you have  any performance gaps or have further questions feel free to open a new issue. 

Thank You
